web/themes/custom/tube_theme/templates/page.html.twig

```diff
@@ -72,17 +71,17 @@
   <div id="page">
     <header id="header" class="header" role="banner" aria-label="{{ 'Site header'|t}}">
       {% block head %}
-        {% if page.top_header_primary or page.top_header or page.top_header_form %}
+        {% if page.top_header_primary or page.top_header_secondary or page.top_header_search %}
           <nav{{ navbar_top_attributes }}>
           {% if container_navbar %}
           <div class="container">
           {% endif %}
               {{ page.top_header_primary }}
-              {{ page.top_header }}
+              {{ page.top_header_secondary }}
               {{ page.site_branding }}
-              {% if page.top_header_form %} <!--search -->
+              {% if page.top_header_search %} <!--search -->
                 <div class="form-inline navbar-form float-right">
-                  {{ page.top_header_form }}
+                  {{ page.top_header_search }}
```

Comment:
It's better for the region names to correspond to the location rather than the contents in case we ever want to place different content in that region. In this case it's fine because we're building this theme for 1 specific project and we know what we're placing in the region, but just as an FYI.